cinema_service/urls.py

```diff
@@ -5,4 +5,5 @@
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path("api/user/", include("user.urls", namespace="user"))
```

Comment:
The `include` function uses the `namespace` argument for the `user` app. Ensure that the `user/urls.py` file defines an `app_name` variable to support this namespacing.